docs/api/using-api.asciidoc

```diff
@@ -34,13 +34,16 @@ For example, the following `curl` command exports a dashboard:
 curl -X POST -u $USER:$PASSWORD "localhost:5601/api/kibana/dashboards/export?dashboard=942dcef0-b2cd-11e8-ad8e-85441f0c2e5c"
 --
 
-The following {kib} APIs are available:
+[float]
+[[api-request-headers]]
+=== Request headers
+
+For all non whitelisted APIs, you must use a header. For information on how to whitelist APIs, refer to the <<settings, `server.xsrf.whitelist`>> documentation.
```

Comment:
I followed the link, and the text for server.xsrf.whitelist says "Definition".  I think we need more information about what whitelisting is about..

Reply:
I know :) Since I'm taking over this PR, I don't have all of the information yet. I was hoping that Larry could fill in some gaps when he can.